### PR TITLE
DS-784: Style top level mega menu links.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Creates a component and poaches styles to allow for a link and not a menu panel in the mega menu top level

# Review By (Date)
- Whenever

# Criticality
- Normal

# Review Tasks

1. Review the [example URL in the ticket description](https://stanford.atlassian.net/browse/DS-784) for an example of the before look
2. Check out this branch
3. Set your storyblok API key on your local to a preview production key
4. Run dev
5. On your localhost navigate to the same URL path as in the example ticket but with `localhost:8000` 
6. Review the `Volunteer` menu item looks like the other items
7. Edit the `isActiveButton` variable in the new `MegaMenuTopLevelLinkItem` component so that it is `true`
8. Review active styles
9. Leave comments

# Associated Issues and/or People
- DS-784

### CODE REVIEW

**Comment by sherakama** [comment]
Build is building https://deploy-preview-906--stanford-alumni.netlify.app 

The failed build status is just the npm package cache that needed to be cleared.

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "gatsby-plugin-netlify": "^4.4.0",
     "gatsby-plugin-postcss": "^5.25.0",
     "gatsby-plugin-react-helmet": "^5.25.0",
-    "gatsby-plugin-robots-txt": "^1.7.0",
+    "gatsby-plugin-robots-txt": "~1.7.0",
```
Comment (sherakama) [inline package.json:31]: This needs to be locked as 1.8 requires gatsby 5.

**Review by ericbakenhus — APPROVED** [review]
LGTM! Matches the existing menu items. 🦖

**Comment by sherakama** [comment]
Gotta look into the cypress fails...

**Comment by sherakama** [comment]
Thats better. Merging into dev to go along with the other updates.

**Pull request merged**